addons/account/models/account_reconcile_model.py

```diff
@@ -569,10 +569,10 @@ def _check_rule_propositions(self, statement_line, candidates):
         if float_is_zero(total_residual - line_residual, precision_rounding=line_currency.rounding):
             return True
 
-        if line_residual > total_residual:
-            amount_percentage = (total_residual / line_residual) * 100
+        if abs(line_residual) > abs(total_residual):
```

Comment:
Most of the time, it's a very bad idea to use absolute values in accounting. In your case, you are adding a new bug when `line_residual` and `total_residual` are not sharing the same sign. I suggest something like:
```
line_residual_to_compare = line_residual if line_residual > 0.0 else -line_residual
total_residual_to_compare = total_residual if line_residual > 0.0 else -total_residual
...
```

Also, adapt your commit message.

Reply:
Your commit should reflect what your are fixing, not the way you are doing it

Reply:
Hello @smetl 

I understand there could be an issue when `line_residual` and `total_residual` do not share the same sign, however, can you please explain how your proposal is different from using `abs` ?

Reply:
@grindtildeath both are managing the sign regarding the statement line sign.

Example:
line_residual = -5
total_residual = 6

Using abs: 5 > 6 is False
Using my suggestion: 5 > -6 is True

Reply:
Ok I missed that you used `line_residual` in the definition of `total_residual_to_compare` and was left wondering :sweat_smile: 
Changing it now :wink: